roles/backup_create_plan/README.md

```diff
@@ -0,0 +1,67 @@
+backup_create_plan
+==================
+
+A role to create a backup plan and optionally a vault. A set of variables for plan rules is included for use as-is or as examples for modification. This role can be combined with the backup_select_resources role to back up a selection of resources.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* backup:CreateBackupVault
+* backup:CreateBackupPlan
+* backup:DescribeBackupVault
+* backup:ListBackupPlans
+* backup:ListTags
+* backup-storage:MountCapsule
+* kms:CreateGrant
+* kms:GenerateDataKey
+* kms:Decrypt
+* kms:RetireGrant
+* kms:DescribeKey
+
+Role Variables
+--------------
+
+* **plan_name**: (Required) The name of the backup plan you want to create
+* **plan_rules**: (Required) A set of rules for the backup, as a list of dicts
```

Comment:
Same here. I will mention that if it is not explicitly set by the user, it defaults to` daily_backup` from what I can see. So it is not really required anymore.

Reply:
It doesn't default to `daily_backup`. That's just provided as a sample rule.

Reply:
Ah, ok. I got confused by the tests https://github.com/redhat-cop/cloud.aws_ops/pull/45/files#diff-0b7bc44f013bf49a0238e1dcd1e9c761c7768dc225cb4e9659103c77b2629474R61